### ISSUE TITLE
`aragon --version` and `aragon --help` shouldn't show an error when not run inside an aragon app project

### ISSUE THREAD

**Issue by izqui** [aragon/aragon-cli#142]
```
$ aragon --version
Error: This directory is not an Aragon project
[ redacted full error message]
4.1.2
```
It should just print `4.1.2` without the error.

**Comment by PascalPrecht** [comment]
Hi @izqui 

I just took a quick look at this one. So at the moment, this happens always when aragon cli is executed outside an aragon project. 

The reason this fails is because of `getTruffleConfig()` inside the `network` option's `coerce()` function, which gets executed synchronously by `yargs` when it bootstraps:

https://github.com/aragon/aragon-cli/blob/d14a980030494388afaf2661a790c309afc06fb6/src/cli.js#L69

`getTruffleConfig()` calls `findProjectRoot()` which errors out

https://github.com/aragon/aragon-cli/blob/d14a980030494388afaf2661a790c309afc06fb6/src/util.js#L11-L21

Turns out `findProjectRoot()` is used all over the place and it's probably reasonable to throw an error in there. However, that'd leave us only with not calling `getTruffleConfig()` in the first place, if we already know we aren't actually interested in any network commands (which is the case when just calling `aragon`, `aragon --help` or `aragon --version`.

`coerce()` will still have to return a value for `network`, so one quick fix would be to bypass `getTruffleConfig()` and simply return `network` in case `help`, `version` (and maybe none)` commands are used.

If this sounds reasonable, I can send a PR for this. Just wasn't sure if you'd be up for this solution, otherwise I would've just proposed it right away. 🙃

Hope this makes sense!

**Comment by izqui** [comment]
Hey @PascalPrecht! Thanks for taking a look at it and detailing your thought process.

I think this issue will be solved with #145, as it was caused by a stupid typo `>= 'version'` should have been `>= 0` 😅  https://github.com/aragon/aragon-cli/pull/145/files#diff-d9d9214113dea1c364abd4ba3c70be11R70

**Comment by PascalPrecht** [comment]
No kidding!

I was honestly wondering what that was about but thought it couldn't be it
cause.. waaay too obvious right?

Ha. Too funny. Thanks for the feedback!
